client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -117,4 +117,5 @@ const MappedDomain = React.createClass( {
 	}
 } );
 
+export { MappedDomain };
```

Comment:
Whoops, you're correct here on the `export { MappedDomain };` syntax.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

The top line linter complains about the `React.createClass` usage if we modify the line 👍  So perfectly fine to avoid here.